docs/Advanced.md

```diff
@@ -68,6 +68,12 @@ lane :calculate do |options|
 end
 ```
 
+## Shell values
+You can get value from shell commands:
+```ruby
+app_version = sh("cd .. && agvtool what-marketing-version -terse1 | tr -d '\n'")
```

Comment:
Please change it to a simpler example since the terminal command itself shouldn't be the focus. 

How about
```sh
pod update
```